src/utils/utils.ts

```diff
@@ -150,3 +150,25 @@ export const sanitizeStringForAttribute = (str: string) => {
   const sanitizedStr = str.replace(/[^a-z0-9]/gi, "-").toLowerCase();
   return sanitizedStr;
 };
+
+/** Provides a random image to use as a placeholder. */
+// export interface GetPlaceholderImageProps {
+//   size?: "small" | "medium" | "large";
+// }
+export const getPlaceholderImage = (size: string = "medium") => {
+  // const { size = "small" } = props;
+  const imageArray = [
+    "https://images.nypl.org/index.php?id=61705",
+    "https://images.nypl.org/index.php?id=61707",
+    "https://images.nypl.org/index.php?id=swope_243048",
+    "https://images.nypl.org/index.php?id=swope_243025",
+    "https://images.nypl.org/index.php?id=swope_244712",
+    "https://images.nypl.org/index.php?id=1252839",
+    "https://images.nypl.org/index.php?id=1252803",
```

Comment:
Can we include these as well?...
- https://images.nypl.org/index.php?id=836241&t=w
- https://images.nypl.org/index.php?id=836235&t=w
- https://images.nypl.org/index.php?id=1587878&t=w
- https://images.nypl.org/index.php?id=1588120&t=w
- https://images.nypl.org/index.php?id=58691098&t=w
- https://images.nypl.org/index.php?id=1203012&t=w